.github/workflows/cli-scripts-deploy-torchserve.yml

```diff
@@ -0,0 +1,32 @@
+name: cli-scripts-deploy-tfserving
+on:
+  schedule:
+    - cron: "0 0/4 * * *"
+  pull_request:
+    branches:
+      - main
+      - cli-preview
+    paths:
+      - cli/deploy-tfserving.sh
```

Comment:
the workflow is not triggering because this does not match the actual filename, please rerun `python readme.py` to update

Reply:
Good catch thanks